submissions/LGeorge007/memory-pair-game/script.js

```diff
@@ -0,0 +1,65 @@
+const allCards = document.querySelectorAll(".flipper");
+const arrOfCards = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19];
+let unopenedCards = arrOfCards.length;
+let firstCard;
+let secondCard;
+let clikCounter = 0;
+
+const checkCard = function (event) {
+    const checkMatch = function () {
+        if (clikCounter == 0) {
+            firstCard = event.path[1];
```

Comment:
Will only comment on this once to shorten the review, you can use `target` to get to the element you need. Also, if you use `.closest()` method you will ensure yourself with a right element, so you don't accidentally add a class to a wrong element, for example.

Reply:
I am creating cards using a documentFragment.
This allows  to have a consistent structure for div class = "flipper". 
And I am referring to the children of the DIV(.flipper), using the following syntax.
firstCard.children[1]
secondCard.children[1]